features/importing/work_import.feature

```diff
@@ -219,46 +219,38 @@ Feature: Import Works
       And I follow "Yes, Continue"
     Then I should see "This work's creator has chosen to moderate comments on the work."
 
-  @work_import_multi_work_backdate
   Scenario: Importing multiple works with backdating
-    When I import the urls
+    When I import the urls with mock websites
```

Comment:
Step [When I import the urls](https://github.com/otwcode/otwarchive/blob/5fbbad0ca130cc382a9593fedf989f9e71ec18fb/features/step_definitions/work_import_steps.rb#L91-L96) doesn't seem to be used anymore. To be deleted?